### DISCUSSION TITLE
Configuring Sflow and Unable to get OIDs

### DISCUSSION THREAD

**Discussion by coestoes** [akvorado/akvorado#1330] [Q&A]
Hello, 
I installed Akvorado in a Ubuntu VM and have configured a router to send Sflow data. My Akvorado.yaml page is configured like this:
networks:
    172.16.0.0/12:
      name: ipv4-customers
      role: customers

With the IP range from which I am receiving Sflows and upon running sudo docker-compose logs -f akvorado-inlet, I receive this error: 
{"level":"error","error":"request timeout (after 1 retries)","exporter":"172.16.11.20","time":"2024-07-30T18:04:26Z","caller":"akvorado/inlet/metadata/provider/snmp/poller.go:115","module":"akvorado/inlet/metadata/provider/snmp","message":"unable to GET (4 OIDs)"}

I have already created a daemon.json file to disable userland-proxy but I am still getting this error and there are no flows on my UI. Any suggestions?

**Comment by vincentbernat** [comment]
Is it the right IP for the exporter (172.16.11.20)?

**Reply by coestoes** [reply]
I should be receiving data from IP ranges 172.16.0.0/12, so yes that is the right IP.

**Reply by vincentbernat** [reply]
Most likely, you "just" need to configure the SNMP community.

**Reply by coestoes** [reply]
I used TCPdump and verified that the VM is receiving sflows. I would still need to configure the SNMP community in the inlet.yaml?

**Reply by vincentbernat** [reply]
Yes, it needs SNMP to retrieve interface names and descriptions as well as the router name.

**Reply by coestoes** [reply]
Will do. I will post any other issues I encounter. I appreciate your help Vincent.
